datadog-logs-oci-orm/logs-function/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Connector Hub function
```

Comment:
logs-function code is a copy paste from Service Connector Hub folder. Only the Dockerfile is new.